pyproject.toml

```
@@ -8,3 +8,8 @@ issue_format = "`#{issue} <https://github.com/pulp/pulp-cli/issues/{issue}>`_"
 
 [tool.black]
 line-length = 100
+
+[tool.isort]
+multi_line_output = 3
+include_trailing_comma = true
```

```suggestion
profile = "black"
multi_line_output = 3
```

The docs say, you should use something like this.